packages/ibm-products/src/components/SteppedAnimatedMedia/SteppedAnimatedMedia.js

```diff
@@ -0,0 +1,153 @@
+/**
+ * Copyright IBM Corp. 2023, 2023
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+// Import portions of React that are needed.
+import React, { useEffect, useRef, useState } from 'react';
+import lottie from 'lottie-web';
+import clamp from 'lodash/clamp';
+// Other standard imports.
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+
+import { getDevtoolsProps } from '../../global/js/utils/devtools';
+import { pkg /*, carbon */ } from '../../settings';
+
+// The block part of our conventional BEM class names (blockClass__E--M).
+const blockClass = `${pkg.prefix}--stepped-animated-media`;
+const componentName = 'SteppedAnimatedMedia';
+
+// NOTE: the component SCSS is not imported here: it is rolled up separately.
+
+// Default values can be included here and then assigned to the prop params,
+// e.g. prop = defaults.prop,
+// This gathers default values together neatly and ensures non-primitive
+// values are initialized early to avoid react making unnecessary re-renders.
+// Note that default values are not required for props that are 'required',
+// nor for props where the component can apply undefined values reasonably.
+// Default values should be provided when the component needs to make a choice
+// or assumption when a prop is not supplied.
+
+// Default values for props
+const defaults = {
+  playStep: 0,
+};
+
+/**
+ * DO NOT USE. This component is for the exclusive use
+ * of other Novice to Pro components.
+ */
```

Comment:
Pretty minor, but we changed this slightly in the Guidebanner and Carousel PR.

https://github.com/carbon-design-system/ibm-products/blob/main_v1/packages/ibm-products/src/components/Carousel/Carousel.js#L40-L42